프로그래머스/1/76501. 음양 더하기/음양 더하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(absolutes, signs) {
+    let arr = []
+    for (let i = 0; i < absolutes.length; i++) {
+        if (signs[i]) {
+            arr.push(absolutes[i])
+        } else {
+            arr.push(-1 * absolutes[i])
+        }
+    }
+    return arr.reduce((acc, cur) => acc + cur, 0);
```

Comment:
```suggestion
return absolutes.reduce((acc, cur, i) => acc + (signs[i] ? cur : -cur), 0);
```
arr라는 새로운 배열을 만들어서 거기에 계산된 값을 저장한 후, reduce를 사용하여 모든 값을 합산하는걸로 보이는데 ! 새 배열을 만들지 않고 바로 합산을 계산할 수 있습니다 ! 참고 한번 부탁드려요 ~

Reply:
:)